tests/unit/core/TestInstallAddon.js

```diff
@@ -231,7 +231,7 @@ describe('withInstallHelpers inner functions', () => {
         guid,
         installURL,
       });
-      const setCurrentStatus = root.prop('setCurrentStatus');
+      const { setCurrentStatus } = root.instance().props;
```

Comment:
Hmmm, it feels like it would be better/easier to use if `root.instance()` was what was returned by `shallowUntilTarget`. That way it feels exactly like regular enzyme `shallow`. Migrating from one to another is easier then and it's less boilerplate. I'm seeing `root.instance()` a lot; that means we should code it out 😄